swan/Dockerfile

```diff
@@ -0,0 +1,162 @@
+    unzip \
+    which \
+   # zeromq3-devel \ -- Does not exist in dnf repos
+    perl-Digest-MD5 \
+    texlive \
+    fontconfig
+
+# Install HEP_OSlibs
+# This installs system packages for LCGs to work
+RUN dnf -y install https://linuxsoft.cern.ch/wlcg/el9/x86_64/wlcg-repo-1.0.0-1.el9.noarch.rpm
+RUN dnf -y install HEP_OSlibs-9.1.0-2.el9
+
+# TODO: Move spark related packages to the third image
+
+# Install required packages
+RUN dnf install -y \
+    # Required by ROOT
+    libXft \
+    # Required by matplotlib
+    tk \
+    # Useful to monitor resource consumption
+    htop \
+    # Install Cloudera dependencies - required by IT Spark clusters
+    alsa-lib \
+    at \
+    bc \
+    cronie \
+    cvs \
+    # db4-cxx \ -- Is not present in DNF repos.
+    # db4-devel \ -- Is not present in DNF repos.
+    file \
+    gdbm-devel \
+    gettext \
+    jpackage-utils \
+    libXtst \
+    man \
+    passwd \
+    # pax \ -- Is not present in DNF repos.
+    # perl-CGI \
+    # perl-ExtUtils-MakeMaker \
+    # perl-Test-Simple \
+    # perl-devel \
+    rsyslog \
+    time \
+    xz-lzma-compat \
+    # Required by Geant4 (libXm)
+    openmotif \
+    # Required by Oracle
+    # TODO: Move to third image: libaio \
+    # Required for key4hep
+    environment-modules && \
+    # Clear the dnf cache as we no longer need to install packages
+    dnf clean all && \
+    rm -rf /var/cache/dnf
+
+# Switch back to jovyan to avoid accidental container runs as root
+USER ${NB_UID}
+
+# Install python pip packages
+RUN mamba install --yes \
+    'jupyter_nbextensions_configurator==0.6.1' \
+    'voila==0.5.5' \
+    'ipyparallel==8.6.1' \
+    # Extensions required by jupyter server
+    'panel==1.3.2' \
+    'webio-jupyter-extension==0.1.0'
+
+# Install TeX Live required packages
+RUN tlmgr install adjustbox \
+    tcolorbox \
+    environ \
+    trimspaces \
+    adjustbox \
+    collectbox \
+    ucs \
+    titling \
+    enumitem \
+    type1cm \
+    cm-super \
+    collection-fontsrecommended
+
+# Required by jupyter-resource-usage
+RUN pip install --no-deps --no-cache-dir psutil==5.8.0
+
+# Install all of our extensions
+# Ignore (almost all) dependencies because they have already been installed or come from CVMFS
+# RUN pip install --no-deps --no-cache-dir \
+# TODO: 3rd image   dask-labextension==6.1.0 \
+#    jupyter-resource-usage==0.6.0 \
+# TODO: 3rd image   hdfsbrowser==1.1.1 \
+# TODO: 3rd image   sparkconnector==2.4.6 \
+# TODO: 3rd image  sparkmonitor==2.1.1 \
+#    swancontents==1.4.2 \
+#    swanhelp==2.0.2 \
+#    swanintro==1.0.0 \
+#    swankernelenv==1.0.0 \
+#    swannotebookviewer==1.2.0 \
+#    swannotifications==1.0.0 \
+#    swanoauthrenew==1.0.1 PyJWT \
+#    swanshare==1.1.1 \
+#    swanportallocator==1.0.1
+            
+# swandask must be installed after its dependency dask-labextension to disable the server extension automatically
+RUN pip install --no-deps --no-cache-dir swandask==0.0.3
+
+ENV SHELL=/bin/bash
```

Comment:
Isn't this already defined in the base image?

Reply:
Yes, removed as well.